### PR TITLE
Adding a few new Sensor features

### PR DESCRIPTION
**Pull request by captainbrosset**
Took a stab at covering Sensor features. Let me know if this organization makes sense.

### CODE REVIEW

File: features/accelerometer.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name: Accelerometer
+description: The `Accelerometer` and `LinearAccelerationSensor` APIs read the acceleration applied to a device in three dimensions, either including the effect of gravity or not, respectively.
+spec: https://w3c.github.io/accelerometer/#accelerometer-interface
+group: sensors
+compat_features:
+  - api.Accelerometer
```
Comment (foolip) [inline features/accelerometer.yml:6]: Here and elsewhere, can you include the interface members too? My starting point for a list would be `npm run traverse | grep Accelerometer` and similar.

File: groups/sensors.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+name: Sensors
```
Comment (foolip) [inline groups/sensors.yml:1]: Can you add a comment to say if the intended scope of the group is only the APIs built on https://w3c.github.io/sensors/, or also older features that are also exposing sensor data, like device motion events?

**Review by Luis-2305 — APPROVED** [review]

File: groups/sensors.yml (hunk 2 above)
Comment (captainbrosset) [reply groups/sensors.yml:1]: No strong opinion on this. I would tend to make this group pretty generic and accommodate all types of features that expose sensor data. We could then perhaps create subgroups for new generic sensor API-based things, and older parts.

File: features/orientation.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+name: Orientation Sensor
```
Comment (foolip) [inline features/orientation.yml:1]:
```suggestion
name: Orientation sensor
```

File: groups/sensors.yml (hunk 2 above)
Comment (foolip) [reply groups/sensors.yml:1]: OK, let's put `device-orientation-events` into this group too then.

File: features/accelerometer.yml (hunk 1 above)
Comment (foolip) [reply features/accelerometer.yml:6]: The constructor is missing. I'll add it.

**Comment by foolip** [comment]
Closing and reopening, see https://github.com/web-platform-dx/web-features/issues/1320.

**Review by foolip — APPROVED** [review]

**Pull request merged**